README.md

```diff
@@ -41,7 +41,7 @@ by teams from Google and IBM, in partnership with the Envoy team at Lyft.
 ## Community meeting
 
 We have public, [recorded](https://www.youtube.com/channel/UC-zVlo1F3mUbExQ96fABWcQ), community meetings.
-They happen on the fourth Thursday of every month at 10am US/Pacific. [Map that to your local time](https://www.google.com/search?q=1000+am+in+pst&hl=en).
+They happen on the fourth Thursday of every month at 10am US/Pacific. [Map that to your local time](https://www.google.com/search?q=1000+am+pst&hl=en).
```

Comment:
Perhaps https://time.is/compare/1000_in_San_Francisco,_California would be a better link?

(searching for PST will be wrong during PDT)